src/MigrationTools/DataContracts/Pipelines/BuildDefinitions.cs

```diff
@@ -356,6 +356,21 @@ public partial class Properties
         public bool CheckoutNestedSubmodules { get; set; }
 
         public long FetchDepth { get; set; }
+        
+        public void SetDefaults() 
```

Comment:
@4damW You could probably assign those default values inside the constructor of this class instead of making an extra method.

Reply:
Yes you're right. I didn't think about that.